### ISSUE TITLE
The results I get with kitti are all red, have you ever encountered this situation?

### ISSUE THREAD

**Issue by yangjin970315** [SysCV/P3Depth#5]
The results I get with kitti are all red, have you ever encountered this situation?

**Comment by wududu123** [comment]
I trained with depth&color  images from tv game gta, it cost 6 days on gtx3090.
The results Looks barely OK，I will continue to train 20 epoch.
You can see the result from this:
https://github.com/md5xwl/myNote/blob/7b8861be4d70c13a7c82087d3f908a2f58d82ece/p3depth_epoch19.jpg

**Comment by BayMaxBHL** [comment]
I tried training on a single GPU and NYU dataset, but the loss didn't change at all. The only change I made was to change the batch size to 8.
I had a gut feeling that the code would work, but I couldn't solve the dilemma I was facing.

**Comment by BayMaxBHL** [comment]
@md5xwl Anything to note when using this training code? It bothered me for a long time.

**Comment by wududu123** [comment]
@BayMaxBHL This is my code https://drive.google.com/file/d/1RRhOknM4tPnWzvi-T0B__BxLWE8famL3/view?usp=sharing.
I training with gta dataset. I add GTA_dataset.py to load color and depth image.
After 40 epochs, the result is bad。 I gave up.

![1668407686539](https://user-images.githubusercontent.com/7579729/201593099-28449d81-50e5-48f1-b9a1-0abfbc033430.jpg)

**Comment by haifengwu205** [comment]
@md5xwl Hello，can you run the test code? I run it with a size mismatch, as follows:
size mismatch for net.coords: copying a param with shape torch.Size([8, 480, 640, 2]) from checkpoint, the shape in current model is torch.Size([16, 480, 640, 2]).

**Comment by yanzilhu** [comment]
> @md5xwl Anything to note when using this training code? It bothered me for a long time.

Hello, did you solve this problem? When I reproduced it, I found that the loss actually increased after training for 20 epochs, which is very unusual.

**Comment by wududu123** [comment]
> > @md5xwl Anything to note when using this training code? It bothered me for a long time.
> 
> Hello, did you solve this problem? When I reproduced it, I found that the loss actually increased after training for 20 epochs, which is very unusual.

????
This project is too old. I'm focused on Depth Anything now.
